lint/lint.go

```diff
@@ -89,7 +93,7 @@ func (l *Linter) lintFileDescriptor(fd *desc.FileDescriptor) (Response, error) {
 	return resp, err
 }
 
-func (l *Linter) runAndRecoverFromPanics(rule ProtoRule, fd *desc.FileDescriptor) (probs []Problem, err error) {
```

Comment:
Why are we making this change? It seems like it _should_ only accept a FileDescriptor.

Reply:
Nevermind. I understand the PR better now.